packages/web/src/components/pages/Tests/TestDetails/TestSettings/SettingsVariables/Arguments.tsx

```diff
@@ -28,6 +28,14 @@ import {prettifyArguments} from '@utils/prettifyArguments';
 
 import {ArgumentsWrapper} from './Arguments.styled';
 
+const formatArgsArray = (args: string[]) => {
+  return args.map(arg => {
+    const trimmedArg = arg.replace(/\s+/g, ' ').trim();
```

Comment:
It should rather not trim the argument, as the argument may start from spaces (rather edge case, but still):
```yaml
args:
- "   i want to have spaced argument  "
```